src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action=" " 
+      method="post" 
+      class="search-form">
```

Comment:
```suggestion
    <form 
      action=" " 
      method="post" 
      class="search-form"
    >
```
fix code style
[checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)